Fixtures/MixedTargets/BasicMixedTargets/MixedTargetWithCustomPaths/Sources/OldCar.m

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+
+// All three import statements should be supported.
+#import "OldCar.h"
+#import "Public/OldCar.h"
+#import "MixedTargetWithCustomPaths/Sources/Public/OldCar.h"
+
+
+// Import the Swift half of the module.
```

Comment:
Should it say "portion" or "part" versus "half"?  Other places as well. I'm not sure since "half" could mean nothing else.

Reply:
Done. I went with "part". Changed here and everywhere.